docker/tester.Dockerfile

```diff
@@ -16,7 +16,7 @@
 
 FROM ubuntu:20.04
 
-ENV DEBIAN_FRONTEND=noninteractive
+ARG DEBIAN_FRONTEND=noninteractive
```

Comment:
Why `ARG` instead of `ENV`? When would one use different value?

Reply:
When using ENV, the variable persists after the build, which is not good if user enters container interactively. When using ARG, the variable is only available during build.
Sources: https://docs.docker.com/engine/faq/#why-is-debian_frontendnoninteractive-discouraged-in-dockerfiles , https://serverfault.com/a/797318